cpu/qn908x/Kconfig.clk

```
@@ -0,0 +1,115 @@
+# Copyright (c) 2020 iosabi
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "QN908x clock configuration"
+    depends on CPU_FAM_QN908X
+
+config BOARD_HAS_XTAL32K
+    bool "Board has an external 32.768 KHz crystal"
```

I think this symbol should not have a prompt, as it is selected and used as interface from the board to indicate that it has a crystal:
```suggestion
    bool
```

---

Thanks, indeed this makes a lot of sense.